### PR TITLE
En 12461 multisigner tests kosk

### PR DESCRIPTION
**Pull request by AdoAdoAdo**
Add the KOSK low level signer into multisigner unit tests.

### CODE REVIEW

File: signing/multisig/blsMultisig_test.go [hunk 1]
```diff
@@ -290,232 +333,447 @@ func TestBLSMultiSigner_CreateOK(t *testing.T) {
...
 
-	err := multiSig.StoreSignatureShare(15, sigShare)
+		err := multiSig.StoreSignatureShare(15, sigShare)
 
-	assert.Equal(t, crypto.ErrIndexOutOfBounds, err)
+		assert.Equal(t, crypto.ErrIndexOutOfBounds, err)
+	})
+	t.Run("with KOSK", func(t *testing.T) {
+		multiSig, _ := multisig.NewBLSMultisig(llSignerKOSK, pubKeys, privKey, kg, ownIndex)
+		sigShare, _ := multiSig.CreateSignatureShare([]byte("message"), []byte(""))
```
Comment (ssd04) [inline signing/multisig/blsMultisig_test.go:659]: can't we use `createSignerAndSigShareBLS` here also?

File: signing/multisig/blsMultisig_test.go [hunk 2]
```diff
@@ -290,232 +333,447 @@ func TestBLSMultiSigner_CreateOK(t *testing.T) {
...
+		_ = multiSig.StoreSignatureShare(ownIndex, sigShare)
+		sigShareRead, err := multiSig.SignatureShare(2)
+
+		assert.Nil(t, sigShareRead)
+		assert.Equal(t, crypto.ErrNilElement, err)
+	})
 }
 
 func TestBLSMultiSigner_SignatureShareOK(t *testing.T) {
 	t.Parallel()
```
Comment (ssd04) [inline signing/multisig/blsMultisig_test.go:753]: `AddSignatureShareOK` from above looks very similar with this test, maybe a duplicate?

File: signing/multisig/blsMultisig_test.go (hunk 1 above)
Comment (AdoAdoAdo) [reply signing/multisig/blsMultisig_test.go:659]: done

File: signing/multisig/blsMultisig_test.go (hunk 2 above)
Comment (AdoAdoAdo) [reply signing/multisig/blsMultisig_test.go:753]: right, removed.

**Review by raduchis — APPROVED** [review]

**Review by ccorcoveanu — APPROVED** [review]

**Pull request merged**